formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -2051,16 +2051,16 @@ and mutuallyRecursiveTwo y => print_int y;
 /* let newMutualRecursionSyntax x => newMutuallyRecursiveTwo (x + x); */
 /* let newMutuallyRecursiveTwo y => print_int y; */
 /*  */
-type x = private int;
+type x = pri int;
 
-type myType 'a 'b 'c = private ('a, 'b, 'c);
+type myType 'a 'b 'c = pri ('a, 'b, 'c);
```

Comment:
I think this should be sufficient enough right?

https://github.com/SanderSpies/reason/blob/4d3b093c1631ff400c8d406a41c799a40d8eb6a9/formatTest/typeCheckedTests/expected_output/mlSyntax.re#L108